.travis.yml

```diff
@@ -0,0 +1,13 @@
+    language: c
+    before_install:
+      - source <(curl -SLs https://raw.githubusercontent.com/adafruit/travis-ci-arduino/master/install.sh)
+    install:
+      - arduino --install-library "ESP32 AnalogWrite"
+      - arduino --install-library "ServoESP32"
+      - arduino --install-library "Smartcar shield"
```

Comment:
I believe just this should be sufficient

Reply:
How do you mean?  To remove everything else? Don´t i need to install the travic-ci-arduino? And the script?

Reply:
No, I mean that you can most likely remove the following two lines:
```yaml
      - arduino --install-library "ESP32 AnalogWrite"
      - arduino --install-library "ServoESP32"
```

Reply:
So I tried it out and the builds fails:
`/home/travis/Arduino/libraries/Smartcar_shield/src/runtime/arduino_runtime/ArduinoRuntime.cpp:7:25: fatal error: analogWrite.h: No such file or directory

compilation terminated.

exit status 1`

Reply:
Oh, that's sad/strange. I thought this command would also install the dependencies too. Oh well, never mind then.